### PR TITLE
Prep for 0.6.2 release

### PR DESCRIPTION
**Pull request by kchawla-pi**

### CODE REVIEW

File: doc/themes/nilearn/layout.html [hunk 1]
```diff
@@ -261,6 +261,8 @@ <h2>Machine learning for Neuro-Imaging in Python</h2>
 
 <h4> News </h4>
   <ul>
+      <li><p><strong>February 2020</strong>: Nilearn 0.6.2 released
+      </p></li>
```
Comment (GaelVaroquaux) [inline doc/themes/nilearn/layout.html]: Can you also remove some of the old ones: let's keep that list no longer than 4 items.

Thanks!

**Comment by GaelVaroquaux** [comment]
Was there a tweet for the release? I missed it and I want to retweet.

**Comment by kchawla-pi** [comment]
Nope. it's your canvas

**Comment by GaelVaroquaux** [comment]
> Nope. it's your canvas

I cannot claim any credit for this work.

**Comment by kchawla-pi** [comment]
My doc building failed , corrupt file, so couldn't update the website. Some
the binder links are a big feature, awaiting to do that first.

**Comment by GaelVaroquaux** [comment]
> My doc building failed , corrupt file, so couldn't update the website. Some the binder links are a big feature, awaiting to do that first.

Sounds reasonable. I'll wait for you to successfully update the website.

Thanks!

**Comment by kchawla-pi** [comment]
My efforts to build and push the new documentation keep failing due to seemingly spurious reasons.
Can either @GaelVaroquaux or @emdupre do it? Le tired.

**Comment by kchawla-pi** [comment]
I will be merging Binh's PR into master. The tag you're looking for to build from is `0.6.2`.

**Comment by emdupre** [comment]
I've built locally and it seems fine. Will upload now -- @GaelVaroquaux can you handle the tweeting ? :bird:

**Comment by kchawla-pi** [comment]
Thanks @emdupre 

Gael insisted he'll handle only the retweeting. We can tweet and retweet each other's.

**Comment by emdupre** [comment]
And now Binder works !!! :tada: :sparkles: :tada: 

I just launched [this](http://nilearn.github.io/auto_examples/plot_nilearn_101.html#sphx-glr-download-auto-examples-plot-nilearn-101-py) and ran the example successfully. I love when it all comes together :heart_eyes:

**Comment by kchawla-pi** [comment]
Its LIVE!! Binder worked as well!

**Comment by GaelVaroquaux** [comment]
> Gael insisted he'll handle only the retweeting.

I feel that the people who do the work deserve credit.

Good job team!

**Comment by emdupre** [comment]
> I feel that the people who do the work deserve credit.

Not sure I understand the relationship, here.

Anyway, I've also tweeted, now: https://twitter.com/emdupre_/status/1232699421844025345?s=20

So I suppose this closed PR is officially closed :laughing: Thanks everyone !

**Pull request merged**